saltproc/openmc_depcode.py

```diff
@@ -1,16 +1,25 @@
 import subprocess
 import os
-import shutil
 import re
 import json
 from pathlib import Path
+import numpy as np
 
+from uncertainties import unumpy
 from pyne import nucname as pyname
 from pyne import serpent
 import openmc
 
 from saltproc import Materialflow
 from saltproc.abc import Depcode
+from openmc.deplete.abc import _SECONDS_PER_DAY
```

Comment:
Okay, you *must* change the name of the abstract file now that you're importing it into other files.

Reply:
This is not the saltproc `abc.py` file, but the openmc one.